client/elife-theme/src/elements/ui/Steps.js

```diff
@@ -39,6 +39,9 @@ export const Steps = {
   `,
 
   Bullet: css`
-    visibility: hidden;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-color: inherit;
+    background-image: url("data:image/svg+xml;utf8,<?xml version='1.0' encoding='UTF-8'?><svg width='10px' height='8px' viewBox='0 0 10 8' version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink'><!-- Generator: Sketch 50.2 (55047) - http://www.bohemiancoding.com/sketch --><title>stepper-tick-icon</title><desc>Created with Sketch.</desc><defs></defs><g id='Symbols' stroke='none' stroke-width='1' fill='none' fill-rule='evenodd'><g id='atoms/step-complete' transform='translate(-5.000000, -6.000000)'><g id='stepper-tick-icon' transform='translate(4.000000, 4.000000)'><rect id='Rectangle' x='0' y='0' width='12' height='12'></rect><polygon id='Shape' fill='#FFFFFF' points='1 6 2.4 4.6 5 7.2 9.6 2.6 11 4 5 10'></polygon></g></g></g></svg>");
```

Comment:
previously we were asked to use icons from here https://material.io/tools/icons/, whereas this looks like it was drawn with sketch...  just checking that is a conscious choice.

Reply:
Fair point ... question for @hdrury1 and/or @NickDuf

Reply:
SVG was from @NickDuf and was drawn in Sketch. Conscious choice.